### PR TITLE
De-register first from SCC or RMT/SMT

### PR DESCRIPTION
**Pull request by keichwa**
- https://github.com/SUSE/spacewalk/issues/23002
- https://bugzilla.suse.com/show_bug.cgi?id=1216946

### CODE REVIEW

File: modules/client-configuration/pages/registration-overview-suse.adoc [hunk 1]
```diff
@@ -19,6 +19,9 @@ You must also have created an activation key.
 For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
 
 
+If the client was registered to SCC, SMT, or RMT previously, remove such a registration with [command]``SUSEConnect`` before migrating the client to {productname} Server.
```
Comment (mcalmer) [inline modules/client-configuration/pages/registration-overview-suse.adoc:22]: I think we should be more explicit and write down the full commands to de-register. Afaik 2 commands are needed. De-register and cleanup. We need to check.

Comment (mcalmer) [reply modules/client-configuration/pages/registration-overview-suse.adoc:22]: https://www.suse.com/de-de/support/kb/doc/?id=000019054

Comment (keichwa) [reply modules/client-configuration/pages/registration-overview-suse.adoc:22]: ok, makes sense.  I'll add the basic commands and add all link to the kb where the user can find all the details (cleanup and verification commands).  If you think we need all the details in our product documentation, I'd create a troubleshooting section in the Admin Guide.

File: modules/client-configuration/pages/registration-overview-suse.adoc [hunk 2]
```diff
@@ -19,9 +19,36 @@ You must also have created an activation key.
...
+
+If the client was registered to SCC, SMT, or RMT previously, remove such a registration with [command]``SUSEConnect`` before migrating the client to {productname} Server.
+If you forget about removing the old registration first, it can happen that repositories not needed anymore getting automatically added to the client even after the client has been migrated to a newer version of a {suse} operating system using {productname}.
+
+* To de-register a {sle} 11 client, on the client, as root, execute:
++
+
+----
+suse_register -E
+----
```
Comment (mcalmer) [inline modules/client-configuration/pages/registration-overview-suse.adoc:43]: I think we do not need SLES11. It is not supported in SUMA (officially) so we should not document anything about it

**Review by mcalmer — CHANGES_REQUESTED** [review]
Just a bit too much added :-)

**Review by mcalmer — APPROVED** [review]

**Pull request merged**